helm-charts/codegen/gaudi-values.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Default values for codegen.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: opea/codegen:latest
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  # tag: "1.0"
+
+port: 6666
+service:
+  type: ClusterIP
+  port: 6666
+
+# To override values in subchart llm-uservice
+llm-uservice:
+  HUGGINGFACEHUB_API_TOKEN: "insert-your-huggingface-token-here"
+  image:
+    repository: opea/llm-tgi:latest
+  service:
+    port: 9000
+# To override values in subchart tgi
+  tgi:
+    # LLM_MODEL_ID: ise-uiuc/Magicoder-S-DS-6.7B
```

Comment:
clean up comments lines

Reply:
Would prefer to keep this comment as examples for user.
Comment line is a different format to specify model.